pyproject.toml

```diff
@@ -23,22 +23,20 @@ classifiers = [
     "Intended Audience :: Developers",
     "Topic :: Software Development :: Libraries :: Python Modules",
     "Operating System :: OS Independent",
-    "Programming Language :: Python :: 2.7",
     "Programming Language :: Python :: 3",
-    "Programming Language :: Python :: 3.5",
-    "Programming Language :: Python :: 3.6",
     "Programming Language :: Python :: 3.7",
     "Programming Language :: Python :: 3.8",
     "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
     "Topic :: Software Development :: Libraries"
 ]
 
 [tool.poetry.dependencies]
-jsonschema = ">=3.2.0, <4.0.0"
-openapi-schema-validator = "<0.2.0"
+python = "^3.7"
+jsonschema = "^4.4.0"
```

Comment:
why jsonschema `4.4.0` is minimal version?

Reply:
This is a leftover as I was running the tests with `poetry` after rebasing with master. It should be `^4.0.0` as in `openapi-schema-validator`. Seem that you changed this already in https://github.com/p1c2u/openapi-spec-validator/pull/143